c_src/membrane_audio_mix_plugin/raw_audio_lib.c

```diff
@@ -0,0 +1,100 @@
+#include "raw_audio_lib.h"
+
+int64_t sample_to_value(unsigned char *sample, CapsAudioRaw *caps)
+{
+    bool format_le = (caps->format & MEMBRANE_SAMPLE_FORMAT_ENDIANITY) == MEMBRANE_SAMPLE_FORMAT_ENDIANITY_LE;
```

Comment:
I think it's a good pattern to call booleans using prefixes like `is_` or `has_`.